simulators/portal-interop/src/main.rs

```diff
@@ -588,9 +588,12 @@ dyn_async! {
             }
         };
 
-        // send a ping so client A sees it as a seen/connected node
-        if let Err(err) = client_a.rpc.ping(target_enr.clone()).await {
-                panic!("Unable to receive pong info: {err:?}");
+        match HistoryNetworkApiClient::add_enr(&client_a.rpc, target_enr.clone()).await {
+            Ok(response) => match response {
+                true => (),
+                false => panic!("AddEnr expected to get true and instead got false")
+            },
+            Err(err) => panic!("{}", &err.to_string()),
```

Comment:
*nit* interpolation simplifies, then some extra context in the error is nice:
```suggestion
            Err(err) => panic!("Failed while trying to add client B's ENR to client A: {err}"),
```